app/models/miq_widget.rb

```diff
@@ -361,7 +361,10 @@ def contents_for_owner(group, user, timezone = nil)
     conditions = {:miq_group_id => group.id}
     conditions[:user_id]   = user.id if user
     conditions[:timezone] = timezone if timezone
-    miq_widget_contents.find_by(conditions)
+    contents = miq_widget_contents.find_by(conditions)
+    conditions.delete(:timezone)
```

Comment:
```ruby
contents = miq_widget_contents.find_by(conditions)

if timezone
  conditions.delete(:timezone)
  contents ||= miq_widget_contents.find_by(conditions)
end

contents
```

Reply:
that avoids doing a find_by in the event timezone is not provided.=

Reply:
👍

Reply:
changed